alea/submitters/htcondor.py

```diff
@@ -123,41 +127,29 @@ def requirements(self):
 
         return _requirements
 
-    def _validate_template_path(self):
-        """Validate the template path."""
-        if self.template_path is None:
-            raise ValueError("Please provide a template path.")
-        # This path must exists locally, and it will be used to stage the input files
-        if not os.path.exists(self.template_path):
-            raise ValueError(f"Path {self.template_path} does not exist.")
-
-        # Printout the template path file structure
-        logger.info("Template path file structure:")
-        for dirpath, dirnames, filenames in os.walk(self.template_path):
-            for filename in filenames:
-                logger.info(f"File: {filename} in {dirpath}")
-        if self._contains_subdirectories(self.template_path):
```

Comment:
`_contains_subdirectories` is no longer used after this PR sop we might as well delete it or am I missing something?